### PR TITLE
Add default request body to fromGlobals

### LINKED ISSUE #15: Should fromGlobals also set the HTTP request body?
This is another thing developers likely want for the ServerRequest object: the request body. PHP puts this away [in `php://input`](https://secure.php.net/manual/en/wrappers.php.php#wrappers.php.input). Currently you would need to [pass a stream](https://github.com/Zegnat/php-website-starter/blob/8b8bf25fd5d83d133f6a49b0f3bb6b9cb197006b/public/index.php#L58) to the `fromArrays` method:

```php
$requestBody = fopen('php://input', 'r');
```

Should `fromGlobals` be extended with something like “defaults to php://input as body”?

### PR DESCRIPTION
**Pull request by Zegnat**
Fixes #15 in both interface and implementation.

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. Could you rebase please?

File: src/ServerRequestCreator.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function fromGlobals(): ServerRequestInterface
         }
         $headers = function_exists('getallheaders') ? getallheaders() : static::getHeadersFromServer($_SERVER);
 
-        return $this->fromArrays($server, $headers, $_COOKIE, $_GET, $_POST, $_FILES);
+        return $this->fromArrays($server, $headers, $_COOKIE, $_GET, $_POST, $_FILES, fopen('php://input', 'r'));
```
Comment (Nyholm) [inline src/ServerRequestCreator.php]: What if `fopen` returns false?

Comment (Nyholm) [reply src/ServerRequestCreator.php]: Great!

**Comment by Zegnat** [comment]
Rebased and a fallback to `null` added in case `php://input` cannot be opened with `fopen`.

**Pull request merged**